src/components/Citizen.tsx

```diff
@@ -0,0 +1,76 @@
+import React, { useEffect, useState } from "react";
+import CitizenData from "./CitizenData";
+import { createClient, WithDefaultsT } from "../generated/definitions/client";
+import { GetBPDCitizenT } from "../generated/definitions/requestTypes";
+import { BPDCitizen } from "../generated/definitions/BPDCitizen"
+import { fromEither, fromLeft, tryCatch } from "fp-ts/lib/TaskEither";
+import { readableReport } from "italia-ts-commons/lib/reporters";
+import { TypeofApiResponse } from "italia-ts-commons/lib/requests";
+
+
+
+function Citizen(props) {
+    const [resultData, setResultdata] = useState<BPDCitizen | undefined>(undefined);
+    const [resultErr, setResulterr] = useState<string>("");
+    
+
+    useEffect(() => {
+      if (props.location.state) {
+        const citizenid = props.location.state.citizenid;
+        // useful to have value also in case of page-refresh
+        window.sessionStorage.setItem('citizenid',props.location.state.citizenid);
+      }
+
+      const simpleClient = createClient({
+        baseUrl: process.env.REACT_APP_API_URL,
+        fetchApi: fetch
+      });
```

Comment:
since you're not configuring the token here, you can export the client as a singleton in its own module:


```ts
// client.ts
export const BackofficeApiClient = createClient({
        baseUrl: process.env.REACT_APP_API_URL,
        fetchApi: fetch
      });
```